collections/includes_value.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Returns whether the given value is part of the collection.
+ *
+ * Example:
+ * ```ts
+ * import { includesValue } from "./includes_value.ts";
+ * import { assert } from "../testing/asserts.ts";
+ *
+ * const input = {
+ *   first: 33,
+ *   second: 34,
+ * };
+ *
+ * assert(includesValue(input, 34));
```

Comment:
I get the idea, but I think sticking with `assertEquals` for consistency and to clearly show the returned value is preferable, especially because the current version works for all truthy values.